src/views/Home/index.js

```diff
@@ -39,25 +39,17 @@ export default class Home extends Component {
 
   async getStatus() {
     const answer = await wellness.getStatus();
-    console.log("this is what i get back: " + answer[0].timestamp)
 
     if(answer.length > 0){
-        if (answer[0].answerValid === true) {
-          this.setState({ answered: true });
-          
-          if(answer[0].userAnswer === true){
-            this.setState({ currentStatus: 'I am symptomatic' });
-          }
-          else{
-            this.setState({ currentStatus: 'I am not symptomatic' });
-          }
-        }
-        if (answer[0].answerValid === false) {
-          this.setState({ answered: false });
-          this.setState({ currentStatus: 'I am symptomatic' });
-        }
-    }
+      if (answer[0].answerValid === true) {
+        this.setState({ answered: true });
```

Comment:
I think one more line is also needed:
     this.setState({ currentStatus: answer[0].userAnswer });

Reply:
Removed current status as the home page is only concerned with whether the student answered the question or not. It renders differently based on the answered State and not the Current Status state. Just some outdated code.